File: app/src/main/java/com/wojdor/hearthcards/application/settings/SettingsActivity.java
package com.wojdor.hearthcards.application.settings;

import android.os.Bundle;
import android.support.annotation.Nullable;

import com.wojdor.hearthcards.R;
import com.wojdor.hearthcards.application.base.BaseActivity;

public class SettingsActivity extends BaseActivity {

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_settings);
    }
}

File: app/src/main/java/com/wojdor/hearthstonecards/application/settings/SettingsActivity.kt
package com.wojdor.hearthstonecards.application.settings

import android.os.Bundle
import com.wojdor.hearthstonecards.R
import com.wojdor.hearthstonecards.application.base.BaseActivity
import kotlinx.android.synthetic.main.activity_settings.*

class SettingsActivity : BaseActivity<SettingsViewModel>() {

    override val viewModelClass = SettingsViewModel::class.java

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_settings)
        initToolbar()
    }

    private fun initToolbar() {
        setSupportActionBar(settingsToolbar)
        supportActionBar?.setDisplayHomeAsUpEnabled(true)
    }

    override fun onSupportNavigateUp(): Boolean {
        onBackPressed()
        return super.onSupportNavigateUp()
    }
}
